docs/interfaces.md

```diff
@@ -357,11 +357,16 @@ Can use kernel tracing facilities. This is restricted because it gives
 privileged access to all processes on the system and should only be used with
 trusted apps.
 
-Usage: reserved
```

Comment:
Oh, nevermind. I see that `Usage: reserved` is removed everywhere. So +1 for this change.